docs/howto/add_pagination_links.md

```diff
@@ -0,0 +1,46 @@
+# How to add pagination links
+
+Pagination links will be included in your response automatically as long as the resource is paginated and if you are using a ```JSON-API``` adapter. The others adapters does not have this feature.
+
+If you want pagination links in your response, use [Kaminari](https://github.com/amatsuda/kaminari) or [WillPaginate](https://github.com/mislav/will_paginate).
+
+```ruby
+#kaminari example
+@posts = Kaminari.paginate_array(Post.all).page(3).per(1)
+render json: @posts
+
+#will_paginate example
+@posts = Post.all.paginate(page: 3, per_page: 1)
+render json: @posts
+```
```

Comment:
@bacarini would you mind updating the will_paginate example to use the ['newer, recommended' syntax that doesn't require the deprecated finders](https://github.com/mislav/will_paginate/issues/322#issuecomment-67785654)?

`@posts = Post.page(3).per_page(1)`

Or even

`@array = Post.all; @posts = WillPaginate::Collection.create(page = 1, WillPaginate.per_page, @array.size) do |pager|  pager.replace @array[pager.offset, pager.per_page].to_a end`